sherpa/cpp_api/bin/online-recognizer.cc

```diff
@@ -68,7 +68,19 @@ To use fast_beam_search with an LG, use
     foo.wav \
     bar.wav
 
-(4) To decode wav.scp
+(4) To use an streaming Zipformer model for recognition
```

Comment:
Could you also update the comment in
https://github.com/k2-fsa/sherpa/blob/master/sherpa/cpp_api/bin/online-recognizer-microphone.cc

Reply:
OK